### PR TITLE
Fix branch

### PR DESCRIPTION
**Pull request by chinmayk1613**
Version update for jQuery_chinmayk1613

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -60,5 +60,9 @@
     "ts-node": "^8.0.3",
     "tslint": "^5.14.0",
     "typescript": "^3.2.4"
-  }
+  },
+  "description": "This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 6.1.3.",
+  "main": "index.js",
```
Comment (janniclas) [inline client/package.json]: why did you add an index.js here?

Comment (chinmayk1613) [reply client/package.json]: I have deleted unnecessary stuff from package-lock.json and client/packag.json

I used npm install to install new version of Jquery, so it includes all the dependency and made changes into the files. Later I understood this is not required as at the time of first run(npm install) all the dependency definition are already there, so need to change that.

**Comment by janniclas** [comment]
I believe this pull request is a duplicate of @Subbup101 . If there are no differences I would ask you to close this one, else highlight the differences please :)

**Comment by chinmayk1613** [comment]
@janniclas .....sure I will close this request..as nothing is new in this request. Thank you for your support,
